src/main/io/spektrum_rssi.h

```diff
@@ -28,6 +28,9 @@
 #define SPEKTRUM_MAX_FADE_PER_SEC       40
 #define SPEKTRUM_FADE_REPORTS_PER_SEC   2
 
+// Number of fade outs counted as a link loss
+#define SPEKTRUM_RSSI_LINK_LOSS_FADES 5
```

Comment:
This does not seem to be used anywhere outside of `spektrum_rssi.c`, so it should be in there.

Reply:
You're right, thanks for the hint. I've moved the definition to spektrum_rssi.c.